app/forms/EducationQuestionnaireForm.scala

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import connectors.exchange._
+import play.api.data.Form
+import play.api.data.Forms._
+import play.api.i18n.Messages
+
+object EducationQuestionnaireForm {
+
+  val form = Form(
+    mapping(
+      "liveInUKBetween14and18" -> Mappings.nonEmptyTrimmedText("error.liveInUKBetween14and18.required", 31),
```

Comment:
Just observed at few places we are converting from "Yes" to "true" and vice versa using pattern match.
Instead if we stick to actual "true" and "false" strings then we can use this Mappings.nonemptyBooleanText formatter and apply simple toBoolean conversion.